index.html

```diff
@@ -4975,6 +4975,621 @@ <h2 id="AudioWorkerParamDescriptor">
+          <dt>
+            Promise&lt;void&gt; import(DOMString moduleUrl)
+          </dt>
+          <dd>
+            <p>
+              Imports a module script returns a promise once it is fully loaded.
+            </p>
+          </dd>
+        </dl>
+
+        <h3>
+          Construction Procedure of AudioWorkletNode and AudioWorkletProcessor
+        </h3>
+        <p>
+          The construction procedure of <a>AudioWokletNode</a> and the
+          corresponding <a>AudioWorkletProcessor</a> object consists of several
+          coordinated interactions and the following algorithm specifies such
+          procedure. Note that these steps must be performed in an atomic
+          fashion.
+        </p>
+        <ol>
+          <li>
+            <em>In the window scope</em>: Start AudioWorkletNode constructor.
+          </li>
+          <li>
+            <em>In the AudioWorkletGlobalScope</em>: Locate a registered
+            AudioWorkletProcessor subclass based on name argument. If none is
+            found, or if subclass does not extend AudioWorkletProcessor, throw
+            an exception from the node constructor.
+            <ol>
+              <li>
+                Start registered constructor for custom processor subclass,
+                passing structured clone of the options object that was provided
+                to AudioWorkletNode.
+              </li>
+              <li>
+                Start super constructor for AudioWorkletProcessor.
+              </li>
+              <li>
+                Initialize the processor’s states.
+              </li>
+              <li>
+                End the construction of an AudioWorkletProcessor and return to
+                AudioWorkletNode constructor.
+              </li>
+            </ol>
+          </li>
+          <li>
+            <em>In the window scope</em>: Construct AudioParam objects in the node
+            based on the information from the processor’s
+            parameterDescriptors getter and initialize parameter values from
+            options where the keys match names declared in
+            AudioParamDescriptors.
+          </li>
+          <li>
+            <em>In the window scope</em>: Return from the AudioWorkletNode
+            constructor.
+          </li>
+        </ol>
+        <section>
+          <h2 id="AudioWorkletNode">
+            The <dfn>AudioWorkletNode</dfn> interface
+          </h2>
+          <p>
+            This interface represens an <a>AudioNode</a> which lives on the main
+            control thread. The user can create this node from an instance of
+            <a>AudioContext</a> and such node can be connected with other built-in
+            <a>AudioNode</a>s to form an audio graph.
+          </p>
+          <dl title=
+          "[Constructor(AudioContext context, DOMString name, optional AudioWorkletNodeOptions options)] interface AudioWorkletNode : AudioNode" 
```

Comment:
Are we explicitly disallowing creating an audio worklet from an offline context?